lua/nvim-treesitter/parsers.lua

```diff
@@ -948,6 +948,26 @@ list.hyprlang = {
   maintainers = { "@luckasRanarison" },
 }
 
+list.idl = {
+  install_info = {
+    url = "https://github.com/cathaysia/tree-sitter-idl",
+    location = "tree-sitter-idl",
+    files = { "src/parser.c" },
+  },
+  maintainers = { "@cathaysa" },
+  experimental = true,
```

Comment:
I'm talking about this: If the parser is not ready for production use, we don't want to include it (yet).